### PR TITLE
New main with all the monitoring on

### PR DESCRIPTION
**Pull request by brunobat**

### CODE REVIEW

File: code/spring-boot-telemetry/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+spring.datasource.driver-class-name=io.opentelemetry.instrumentation.jdbc.OpenTelemetryDriver
 
-spring.datasource.url=jdbc:postgresql://db:5432/demo
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto=create
 
 spring.datasource.username=demo
 spring.datasource.password=demo
 
 spring.jpa.hibernate.ddl-auto=update
```
Comment (jeanbisutti) [inline code/spring-boot-telemetry/src/main/resources/application.properties:12]: To have the same configuration for Quarkus and Spring:

```suggestion
spring.jpa.hibernate.ddl-auto=drop-and-create
```

File: code/spring-boot-telemetry/src/main/resources/application.properties [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 spring.jpa.open-in-view=false
 
+applicationinsights.connection.string=InstrumentationKey=yourInsightsKey
```
Comment (jeanbisutti) [inline code/spring-boot-telemetry/src/main/resources/application.properties:15]: To have the same thing for Spring and Quarkus:

```suggestion
applicationinsights.connection.string=InstrumentationKey=00000000-0000-0000-0000-000000000000
```

**Review by jeanbisutti — APPROVED** [review]

File: code/quarkus-telemetry/quarkus-telemetry-superhero/docker-compose.yml [hunk 3]
```diff
@@ -14,14 +14,17 @@ services:
       retries: 5
 
   superhero:
-    image: localhost/quarkus-telemetry-superhero:latest
```
Comment (SandraAhlgrimm) [inline code/quarkus-telemetry/quarkus-telemetry-superhero/docker-compose.yml:17]: I'd keep localhost here as we can't rely on the correct version in the registry

File: .github/workflows/quarkus-native-monitoring-AutoDeployTrigger-cd1d418e-9e7d-4855-bcfa-dc4948f26827.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        uses: azure/container-apps-deploy-action@v2
+        with:
+          appSourcePath: ${{ github.workspace }}/code/quarkus-telemetry/quarkus-telemetry-superhero/
+          registryUrl: nativetelemetry.azurecr.io
+          registryUsername: ${{ secrets.QUARKUSNATIVEMONITORING_REGISTRY_USERNAME }}
+          registryPassword: ${{ secrets.QUARKUSNATIVEMONITORING_REGISTRY_PASSWORD }}
+          containerAppName: quarkus-native-monitoring
+          resourceGroup: sakriema-graalvm
+          imageToBuild: nativetelemetry.azurecr.io/quarkus-native-monitoring:${{ github.sha }}
+          environmentVariables: 'QUARKUS_DATASOURCE_JDBC_URL=${{ secrets.DATABASE_URL_SUPER }} QUARKUS_DATASOURCE_USERNAME=${{ secrets.DATABASE_USERNAME }} QUARKUS_DATASOURCE_PASSWORD=${{ secrets.DATABASE_PASSWORD }} QUARKUS_OTEL_AZURE_APPLICATIONINSIGHTS_CONNECTION_STRING=${{ secrets.APPLICATIONINSIGHTS_CONNECTION_STRING }}'
```
Comment (SandraAhlgrimm) [inline .github/workflows/quarkus-native-monitoring-AutoDeployTrigger-cd1d418e-9e7d-4855-bcfa-dc4948f26827.yml:44]: let's rename to AZURE_APPLICATIONINSIGHTS_CONNECTION_STRING for ease

File: code/spring-boot-telemetry/src/main/resources/application.properties [hunk 5]
```diff
@@ -1,15 +1,17 @@
...
 
-spring.datasource.url=jdbc:postgresql://db:5432/demo
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto=create
 
 spring.datasource.username=demo
 spring.datasource.password=demo
 
-spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.hibernate.ddl-auto=drop-and-create
```
Comment (SandraAhlgrimm) [inline code/spring-boot-telemetry/src/main/resources/application.properties:12]: remove doubled spring.jpa.hibernate.ddl-auto

**Review by SandraAhlgrimm — CHANGES_REQUESTED** [review]
small adjustments needed as commented

**Review by SandraAhlgrimm — APPROVED** [review]

**Pull request merged**